verification/src/config.rs

```diff
@@ -25,16 +26,23 @@ impl Default for ServerConfiguration {
     }
 }
 
+#[derive(Default, Deserialize, Clone)]
+pub struct VerifierConfiguration {
+    pub disabled: bool,
```

Comment:
I'm not sure about this. I don't see much a reason to disable some functionality from the service in real world usage
We mostly want to disable some services for tests, where we can do it by code, and moving it into config? I'm not sure

Reply:
Actually, your way is worse, because we'll still need parameters for disabled services. Like, Sourcify requieres `SourcifyConfiguration` for it, but if we want to start only `verifier`, then we don't need this config
I thought about making a `new` function for all the services, which initializes all the children (and will be used in main), and may be an `empty` function, which doesn't initialize any children, with functions like `with_service_foo(service)`, which will init the `foo` child service
So it'll look like
```rust
let service = AppRoute::empty(config).with_verification(Verification::new().await.unwrap());
```
notice no sourcify config needed here

Reply:
> I'm not sure about this. I don't see much a reason to disable some functionality from the service in real world usage
We mostly want to disable some services for tests, where we can do it by code, and moving it into config? I'm not sure

I believe it may be a useful feature actually. Some may wish to disable sourcify proxy and use only internal verifier, for example which seems quite possible.

> Actually, your way is worse, because we'll still need parameters for disabled services.

Reply:
> It does, but is it a problem actually?
Right now it isn't, but if a service will accept something long to load. For example, imagine if `VerificationRoute` would accept `GithubFetcher` as an argument, then we'll need to load it

And actually, speaking of which, I think there may be another way to make optional services

Reply:
Do you have any idea how what we can do with that while leaving services disabling feature?

In general, we can overcome the problem by making `VerifierConfiguration` (`SourcifyConfiguration`) fields optional, and adding another structure which is obtained from the former and which sets absent fields with default values. But it requires another level of indirection and probably is better to implement only if current solution becomes too inefficient. If you have better idea we may try to implement it instead.